src/supermin-init-prelude.sh

```diff
@@ -16,13 +16,14 @@ LANG=C /sbin/load_policy  -i
 /usr/sbin/dhclient eth0
 
 # set up workdir
-mkdir -p "${workdir:?}"
+mkdir -p "${workdir:?}" /usr/lib/coreos-assembler
 mount -t 9p -o rw,trans=virtio,version=9p2000.L workdir "${workdir}"
+mount -t 9p -o rw,trans=virtio,version=9p2000.L cosa /usr/lib/coreos-assembler
```

Comment:
i think we can get around and not need this 9p mount by just doing something like: 

```
[dustymabe@media src (PR360 *%)]$ git diff
diff --git a/src/cmdlib.sh b/src/cmdlib.sh
index 5e7ce10..b94253e 100755
--- a/src/cmdlib.sh
+++ b/src/cmdlib.sh
@@ -266,6 +266,9 @@ runvm() {
     while IFS= read -r dep; do rpms+="$dep "; done < <(grep -v '^#' "${DIR}"/vmdeps.txt)
     # shellcheck disable=SC2086
     supermin --prepare --use-installed -o "${vmpreparedir}" $rpms
+    # Add cosa source files to supermine appliance
+    find /usr/lib/coreos-assembler/ > "${vmpreparedir}/hostfiles"
+
 
     # the reason we do a heredoc here is so that the var substition takes
     # place immediately instead of having to proxy them through to the VM
```

Reply:
Worth a try.  Is there a reason a 9p mount is not a good idea?

Reply:
Tested and repushed with the change.

Reply:
the above lines are no longer needed?